code/modules/flufftext/Dreaming.dm

```diff
@@ -19,7 +19,8 @@ mob/living/carbon/proc/dream()
 		"a vox","a plasmaman","a skellington","a diona","the derelict","the end of the world","the thunderdome","a ship full of dead clowns","a chicken with godlike powers",
 		"a red bus that drives through space","an alien artifact","the mechanic","a newspaper","an insectoid","a slime","a slime person","a mushroom person",
 		"the Cult of Nar-Sie","the Wizard Federation","an impossibly gigantic lamprey floating through space, bending reality as it goes","a sword that talks",
-		"an eclipse","a sandwich so tall that it pierces the heavens","attack ships on fire off the shoulder of Orion","C-beams glittering in the dark near the Tannhäuser Gate"
+		"an eclipse","a sandwich so tall that it pierces the heavens","attack ships on fire off the shoulder of Orion","C-beams glittering in the dark near the Tannhäuser Gate",
+		"and exploding computer",
```

Comment:
Huh?

Reply:
Oh yeah when my Windows died last week I managed to get git set back up a few minutes before midnight and wanted to make a quick commit to keep my combo. But turns out that while commits don't count as "contributions". I should have opened an issue about one of the countless bugs that aren't yet in the database. So now I have this blank spot of shame 
![image](https://user-images.githubusercontent.com/7573912/124410517-f274ee80-dd4a-11eb-96ca-7806170097c3.png)

Reply:
OK so because you wanted to see a green square, you unatomically added a message without noting this anywhere, which also happened to contain a typo?

Reply:
Yep, that was a bad move on all accounts.

Although regarding `without noting this anywhere` I routinely update this .dm to force github's linters to re-run when they fail for no reason. Kurfurst and others know I usually do that and although the reason this time was a bad one I think updates to dreaming.dm are fluff enough to not warrant a changelog entry.

Reply:
But I'll revert that commit for sure.